components/director/cmd/director/main.go

```diff
@@ -96,6 +96,8 @@ type config struct {
 	OAuth20      oauth20.Config
 
 	Features features.Config
+
+	ProtectedLabelPattern string `envconfig:"default=.*_defaultEventing"`
```

Comment:
Sure, multiple DB requests may be necessary (as you point out there is no "OR" operator in the storage layer) once we have multiple protected labels, but it will give us more flexibility going forward.

Reply:
Actually with one regex you can also match different single string values. I think this will cover the scenarios you mention.
Do you have any suggestion on how to implement the multiple protected labels without having the OR sql query builder?

Reply:
Oh, I think I got you. With regex we could say something like "(.*_defaultEventing)|(.someOtherPattern)" (yeah, I know my regex example and syntax probably sucks, but the idea is more important). And we could add more protected labels basically by means of regex syntax?
Anyways, I looked at the `labelRepo.DeleteByKeyNotPattern()` implementation - yeah, no sure I can suggest a clean proposal right now.. it would involve multiple DB requests for sure with the current storage layer.